### PR TITLE
Feature/fix develop ui

### PR DESCRIPTION
**Pull request by basil7vals**

### CODE REVIEW

**Comment by basil7vals** [comment]
Cycle - 0 reviewed

**Comment by abdullah-siddiqui-git** [comment]
Cycle - 1 reviewed.

File: app/assets/javascripts/custom.js [hunk 1]
```diff
@@ -1,7 +1,14 @@
 // Menu Toggle Script
 $(function() {
-  $("#js_side_menu_toggle_button").click(function(e) {
-      e.preventDefault();
+  $("#js_side_menu_toggle_button").click(function(event) {
+      event.preventDefault();
       $("#wrapper").toggleClass("toggled");
   });
+
+  $('input').filestyle({
```
Comment (basil7vals) [inline app/assets/javascripts/custom.js]: assign proper id

File: app/helpers/application_helper.rb [hunk 2]
```diff
@@ -1,2 +1,5 @@
 module ApplicationHelper
+  def container_class
```
Comment (basil7vals) [inline app/helpers/application_helper.rb]: main_layout_class

File: app/views/documents/_show_documents.html.erb [hunk 3]
```diff
@@ -1,18 +1,20 @@
...
+    <% else %>
+      <td>Document</td>
+    <% end %>
+
+    <td><%= index_no %></td>
+    <td><%= document_type.document_file_name %></td>
+    <td><%= document_type.document_content_type %></td>
+    <td><%= link_to "Open", document_type.document.url(:original, false), target: :_blank %></td>
+
+    <% if @document_type == 'Issue' %>
```
Comment (basil7vals) [inline app/views/documents/_show_documents.html.erb]: move this logic to model

File: app/views/user_companies/find_user_by_email_response.js.erb [hunk 4]
```diff
@@ -1,9 +1,7 @@
...
-    $(location).attr("href", '<%= new_user_session_url(subdomain: @found_users.first.company.subdomain, user_email: @found_users.first.email) %>');
-  <% else %>
-    $(location).attr("href", '<%= show_companies_path(user_email: @found_users.first.email) %>');
-  <% end %>
-});
\ No newline at end of file
+<% if @found_users.count == 0 %>
+  alert("Invalid Email.");
+<% elsif @found_users.count == 1  %>
+  $(location).attr("href", '<%= new_user_session_url(subdomain: @found_users.first.company.subdomain, user_email: @found_users.first.email) %>');
```
Comment (basil7vals) [inline app/views/user_companies/find_user_by_email_response.js.erb]: Verify

**Pull request merged**